docs/installation/server-config/security.rst

```diff
@@ -22,6 +25,83 @@ file and then :ref:`start the KSQL server <start_ksql-server>` with your configu
 .. contents:: Table of Contents
     :local:
 
+Configuring KSQL for BASIC HTTP Authentication
+----------------------------------------------
+KSQL can be configured to require users to authenticate using a username and password via the BASIC
+HTTP authentication mechanism.
+
+NOTE: If using BASIC authentication it is highly recommended that you configure the KSQL server to
```

Comment:
Use `.. note:: ` syntax.

Reply:
Not: BASIC authentication -> basic authentication

Reply:
done.